src/containerapp/azext_containerapp/_validators.py

```diff
@@ -0,0 +1,94 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from unicodedata import name
+from azure.cli.core.azclierror import (ValidationError, RequiredArgumentMissingError)
+
+
+def example_name_or_id_validator(cmd, namespace):
```

Comment:
Don't see this file or validations being used anywhere? Can you point me to where this is used?

Reply:
These validators are used in the _params.py file, core CLI runs them to validate the input at the beginning of execution